### PR TITLE
boards: arm: sensortile_box_pro: add bluenrg-lp chip description

### PR DESCRIPTION
**Pull request by avisconti**
Extend sensortile_box_pro document adding BlueNRG-LP chip description.

This PR replaces #74255

### CODE REVIEW

File: boards/st/sensortile_box_pro/doc/index.rst [hunk 1]
```diff
@@ -212,6 +215,14 @@ The final application may use it to declare SensorTile.box PRO device as belongi
 certain standard or vendor class, e.g. a CDC, a mass storage or a composite device with both
 functions.
 
+BlueNRG-LP chip
+===============
+
+The board is equipped with an STMicroelectronics `BlueNRG-LP`_ chip. Before using the board Bluetooth
+functionality, a firmware upgrade of the chip may be necessary, in order to run a firmware which is
+compatible with the Zephyr BT stack (see `stsw-mkbox-bleco`_ for more information).
+The upgrade may be easily performed following the procedure explained in `ble-fw-upg`_.
```
Comment (erwango) [inline boards/st/sensortile_box_pro/doc/index.rst:224]:
```suggestion
The board is equipped with an STMicroelectronics `BlueNRG-LP`_ chip. Before running Zephyr Bluetooth samples
on SensorTile.box PRO, it is required to upgrade the BlueNRG chip with a Zephyr BLE stack compatible firmware. 
The upgrade may be easily performed using the application provided in `SensorTile.box PRO BLE firmware upgrade package`_.
For more information about BLE binaries for SensorTile.box family, see `stsw-mkbox-bleco`_.
```

**Review by erwango — DISMISSED** [review]
LGTM, but see comments for a rewording proposal.

**Comment by avisconti** [comment]
> LGTM, but see comments for a rewording proposal.

ok, seems better

**Comment by avisconti** [comment]
@erwango can you possibly review it again? I just applied your suggestions

**Review by erwango — APPROVED** [review]

**Comment by avisconti** [comment]
Added also @kartben and @fabiobaltieri in the reviewer list. If you can also give a look to this PR... it is only a small documentation change

**Review by fabiobaltieri — APPROVED** [review]

**Pull request merged**